src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.Helpers.cs

```diff
@@ -90,31 +90,25 @@ private static string WriteCoreString(object value, Type type, JsonSerializerOpt
             return result;
         }
 
-        private static string WriteValueCore(Utf8JsonWriter writer, object value, Type type, JsonSerializerOptions options)
+        private static void WriteValueCore(Utf8JsonWriter writer, object value, Type type, JsonSerializerOptions options)
         {
             if (options == null)
             {
                 options = JsonSerializerOptions.s_defaultOptions;
             }
 
-            string result;
-
-            using (var output = new PooledByteBufferWriter(options.DefaultBufferSize))
```

Comment:
This was dead code.

Reply:
So, this was just pure overhead that was redundant/unnecessary work?